addons/web/static/src/webclient/clickbot/clickbot.js

```diff
@@ -255,49 +257,56 @@
+            ".o_control_panel .o_dropdown_container.o_filter_menu"
         );
+        // Is there a filter menu in the search bar
+        if (!filterMenuButton) {
+            return;
+        }
 
-        const simpleFilterSel = ".o_control_panel .o_filter_menu > .dropdown-menu > .dropdown-item";
-        const dateFilterSel = ".o_control_panel .o_filter_menu > .dropdown-menu > .dropdown";
+        const simpleFilterSel = ".o_control_panel .o_filter_menu > .dropdown-item";
+        const dateFilterSel = ".o_control_panel .o_filter_menu > .o_accordion";
         const filterMenuItems = document.querySelectorAll(`${simpleFilterSel},${dateFilterSel}`);
         console.log("Testing", filterMenuItems.length, "filters");
         for (const filter of filterMenuItems) {
             const currentViewCount = viewUpdateCount;
-            if (filter.classList.contains("dropdown")) {
+            if (filter.classList.contains("o_accordion")) {
                 await triggerClick(
-                    filter.querySelector(".dropdown-toggle"),
+                    filter.querySelector(".o_accordion_toggle"),
                     `filter "${filter.innerText.trim()}"`
                 );
                 // the sub-dropdown opens 200ms after the mousenter, so we trigger an ArrayRight
                 // keydown s.t. it opens directly
-                window.dispatchEvent(new KeyboardEvent("keydown", { key: "ArrowRight" }));
-                await waitForNextAnimationFrame();
+                await waitForNextAnimationFrame(); // JPP : Not sure this is needed anymore
 
                 // If a fitler has options, it will simply unfold and show all options.
                 // We then click on the first one.
-                const firstOption = filter.querySelector(".dropdown-menu > .dropdown-item");
+                const firstOption = filter.querySelector(
+                    ".o_accordion > .o_accordion_values > .dropdown-item"
+                );
                 if (firstOption) {
                     await triggerClick(
                         firstOption,
                         `filter option "${firstOption.innerText.trim()}"`
                     );
                     await waitForCondition(() => currentViewCount !== viewUpdateCount);
                 }
-            } else if (!filter.classList.contains("o_menu_item")) {
-                // Advanced Search item
-                await triggerClick(filter, `Advanced Search}"`);
+            } else if (filter.classList.contains("o_add_custom_filter")) {
+                // Add Custom Filter item
+                await triggerClick(filter, `Add Custom Filter"`);
                 await waitForCondition(() => !!document.querySelector(`.modal button.btn-close`));
                 await triggerClick(
                     document.querySelector(`.modal button.btn-close`),
-                    "close 'Advanced Search' dialog"
+                    "close 'Add Custom Filter' dialog"
```

Comment:
@aab-odoo Do you think it's normal to just open the 'Add Custom Filter' without adding a new filter ?